xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/historyinline.vm

```diff
@@ -158,10 +158,12 @@ $xwiki.ssfx.use('uicomponents/pagination/pagination.css', true)##
         #if ($xwiki.hasMinorEdit())
           #if ("$!request.showminor" != 'true')
             <input class="btn btn-default" type="submit" name="viewminorversions" value="$services.localization.render('core.viewers.history.showMinorEdits')"
-              onclick="document.forms.${formname}.action='$tdoc.getURL('view', "viewer=history&amp;showminor=true&amp;$docvariant")'; if (document.forms.${formname}.onsubmit) document.forms.${formname}.onsubmit();" />
+              onclick="document.forms.${formname}.action='$tdoc.getURL('view', "$docvariant")'; if (document.forms.${formname}.onsubmit) document.forms.${formname}.onsubmit();" />
```

Comment:
Could you explain why you removed "viewer=history"? And why you removed "showminor=1"? Maybe it's obvious but since I don't know this code, I'm curious to know why changing the computed URL in this way makes it work? Thx

Reply:
@vmassol I did that for the same reason I added a hidden input for viewer param, when the request is changed to GET, with everything else being kept the same, providing params in the action field doesn't work and hidden inputs need to be added.